csm_web/scheduler/views/matcher.py

```diff
@@ -520,21 +518,24 @@ def run_matcher(course: Course):
         - unmatched: [int, int, ...] of mentor ids
     """
     # get slot information
-    slots = MatcherSlot.objects.filter(matcher=course.matcher)
+    matcher_slots = MatcherSlot.objects.filter(matcher=course.matcher)
     # get preference information
-    preferences = MatcherPreference.objects.filter(slot__matcher=course.matcher)
```

Comment:
This should be `matcher_preferences` (plural) to match the other variables (and the semantics of this variable)